apollo-portal/src/main/resources/static/config.html

```diff
@@ -88,6 +88,10 @@
                                         <th>{{'Common.AppOwner' | translate }}:</th>
```

Comment:
How about we append the user name here, just like what we did for orgInfo?

Reply:
BTW, it seems the preferred user name is only used in the project info and the upper-right user info, shall we also use them in other places?